src/components/ratings.js

```diff
@@ -1,44 +1,48 @@
 import React, { Component } from 'react';
-import {connect} from 'react-redux';
-
+import { connect } from 'react-redux';
 import Ratings from 'react-ratings-declarative';
 
 import { updateBeer } from '../actions/index';
 
 export var ratingChange = false;
 
 class RatingsComponent extends Component {
-    constructor(props) {
-        super(props);
-
-        this.state = {
-            rating: props.beerRating
-        }
-
-        this.changeRating = this.changeRating.bind(this);
-    }
-
-    changeRating(id, rating){
-        this.props.updateBeer(id, rating);
- ratingChange = true;
-        this.setState({
-            rating: rating
-        })
-    }
-
-    render() {
-        return (
-            <Ratings rating={this.state.rating}
-                widgetRatedColors="blue"
-                changeRating={(rating) => { this.changeRating(this.props.beerId, rating) }}>
-                <Ratings.Widget />
-                <Ratings.Widget />
-                <Ratings.Widget />
-                <Ratings.Widget />
-                <Ratings.Widget />
-            </Ratings>
-        )
-    }
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      rating: props.beerRating,
+    };
+  }
+
+  changeRating = (id, rating) => {
+    this.props.updateBeer(id, rating);
+    ratingChange = true;
```

Comment:
use state or redux

Reply:
the update method is going  through actions, followed by an update of the component state.